tests/static/assets/webxr-cube.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1"><title>WebXR Cube</title></head><body> <script type=module>var parcelRequire=function(e,r,t,n){var i,o="function"==typeof parcelRequire&&parcelRequire,u="function"==typeof require&&require;function f(t,n){if(!r[t]){if(!e[t]){var i="function"==typeof parcelRequire&&parcelRequire;if(!n&&i)return i(t,!0);if(o)return o(t,!0);if(u&&"string"==typeof t)return u(t);var c=new Error("Cannot find module '"+t+"'");throw c.code="MODULE_NOT_FOUND",c}p.resolve=function(r){return e[t][1][r]||r},p.cache={};var l=r[t]=new f.Module(t);e[t][0].call(l.exports,p,l,l.exports,this)}return r[t].exports;function p(e){return f(p.resolve(e))}}f.isParcelRequire=!0,f.Module=function(e){this.id=e,this.bundle=f,this.exports={}},f.modules=e,f.cache=r,f.parent=o,f.register=function(r,t){e[r]=[function(e,r){r.exports=t},{}]};for(var c=0;c<t.length;c++)try{f(t[c])}catch(e){i||(i=e)}if(t.length){var l=f(t[t.length-1]);"object"==typeof exports&&"undefined"!=typeof module?module.exports=l:"function"==typeof define&&define.amd?define(function(){return l}):n&&(this[n]=l)}if(parcelRequire=f,i)throw i;return f}({"noWu":[function(require,module,exports) {
```

Comment:
Why is this file parcelified? Is it being uploaded somewhere?

I could understand if this was being uploaded, in which case file size matters so we don't waste bandwidth. But otherwise I think we should stick to unminified test code.

Reply:
It's not being uploaded in this test case, but my thinking was that we could use this single `.wbn` asset for other tests that might be uploading stuff rather than creating another another one that's minified.

I also parcelified it because it's easier to write tests that package WebXR apps with a single file like `compileFromFile` when packaging a single file (https://github.com/webaverse/xrpackage/blob/4a044b7858f43d50921b60e051ea8945b5c7e4e8/tests/compile-from-file.js#L9)

Reply:
Hm, in that case could we parcel it without this minification? Basically it's impossible to edit this HTML in this form.

Reply:
I just pushed a change to unminify the JS and HTML!